assets/sass/patterns/organisms/content-aside.scss

```diff
@@ -14,24 +14,21 @@
 
   @media only screen and (min-width: #{get-rem-from-px($bkpt-site--extended-wide)}em) {
     @include padding(86, "top");
+    @include padding(4, "right");
     align-items: flex-end;
     align-self: flex-start;
     display: flex;
     flex-direction: column;
+    max-height: 100vh;
+    overflow-y: auto;
+    position: sticky;
+    top: -14px;
```

Comment:
Should this be -14px?

Reply:
We need 86px from the top on load and then we need 72px when we are scrolling down. I removed the JS so I used that negative value to achieve the correct distance.

Reply:
Ok.. Then I am happy with this